vmm/src/vmm_config/drive.rs

```diff
@@ -158,72 +157,84 @@ impl BlockDeviceConfigs {
     }
 
     fn create(&mut self, block_device_config: BlockDeviceConfig) -> Result<()> {
-        // check if the path exists
+        // If no path exists, then an error is returned.
         if !block_device_config.path_on_host.exists() {
-            return Err(DriveError::InvalidBlockDevicePath);
+            Err(DriveError::InvalidBlockDevicePath)
         }
-
-        if self
+        // If the path already exists, then an error is returned.
```

Comment:
In the context of the previous comment, this comment doesn't really make sense.
Suggestion: nit: Check if a device with the same path already exists.